lib/learn_test.rb

```diff
@@ -25,6 +25,7 @@
 require_relative 'learn_test/strategies/csharp_nunit'
```

Comment:
Instead of commenting out all of the old Python files, can we just comment out line 20 of `lib/learn_test.rb`?

Commenting out entire files wreaks havoc on the Git history

Reply:
great suggestion thanks gabe

Reply:
fixed @gj